--- a/src/caelestia/subcommands/record.py
+++ b/src/caelestia/subcommands/record.py
@@ -55,10 +55,10 @@
                 if self.intersects((monitor["x"], monitor["y"], monitor["width"], monitor["height"]), r):
                     rr = round(monitor["refreshRate"])
                     max_rr = max(max_rr, rr)
-            args += ["-f", str(max_rr)]
+            args += ["-f", str(min(60, max_rr))]
         else:
             focused_monitor = next(monitor for monitor in monitors if monitor["focused"])
             if focused_monitor:
-                args += [focused_monitor["name"], "-f", str(round(focused_monitor["refreshRate"]))]
+                args += [focused_monitor["name"], "-f", str(min(60, round(focused_monitor["refreshRate"])))]

         if self.args.sound:
             args += ["-a", "default_output"]
